src/include/kdbmount.h

```diff
@@ -0,0 +1,49 @@
+/* Mount API for newbackend, currently re-implementation of old mounting process in C instead of C++ */
+
+#ifndef KDB_MOUNT_H
+#define KDB_MOUNT_H
+
+#include <stdbool.h>
+#include <kdb.h>
+
+
+struct cBackendInfo {
+	/* where the backend is mounted */
+	const char * mountpoint;
+	/* The configuration file path to this backend */
+	const char * path;
+};
+
+/* Linked List for BackendInfo structs */
+struct cListBackendInfo {
+	struct cBackendInfo backendInfo;
+	struct cListBackendInfo * next;
+};
+
+
+/* Actual mounting functionality */
+
+/* from mountbase.cpp */
+const KeySet * cReadMountConf (bool clNull, bool clFirst, bool clSecond, bool clThird, bool clVerbose, bool clDebug);
```

Comment:
Is this public API? Then it should start with a common prefix, e.g. elektraMount.